shared/scales.ts

```diff
@@ -28,5 +28,9 @@ export const SCALES: { [id in ScaleName]: { name: string; values: Array<CardValu
     name: 'Fixed Ratio',
     values: ['1', '2', '4', '8', '16', '32', '64', '128', ...SPECIAL_VALUES_ORDERED],
   },
+  LINEAR_SCALE: {
```

Comment:
I, personally, would tend to put the linear scale at the bottom, since it is the least common scale.

Reply:
This makes sense. I had a similar thought and therefore put it at the bottom of the scales containing numbers. The "SIZES" scale felt different because it consists of letters - I assumed that dissecting the number scales with a letters scale is less intuitive but I'm happy to change it. Is there any research on this?